### PR TITLE
[dist] More backend/frontend features

### PR DESCRIPTION
**Pull request by hennevogel**
Reading thousands of commit messages is fun. Let's do this again!

### CODE REVIEW

File: ReleaseNotes-2.7 [hunk 1]
```diff
@@ -3,63 +3,126 @@
...
+* Support for simple image creation
+* New hardware architectures are supported: k1om, aarch64_ilp32
+* Many improvements in the maintenance support features
+* Improved scheduler speed
+* Allow non-obs urls in kiwi, map to obs urls if possible
+* collax package format support
+* Support virt-builder repository meta data
+* bs_admin:
+  * give the admin a chance to prefer publish events with --prefer-publish-event
+  * --vm-enable-console
```
Comment (adrianschroeter) [inline ReleaseNotes-2.7]: that is a worker feature.

File: ReleaseNotes-2.7 [hunk 2]
```diff
@@ -3,63 +3,126 @@
...
+operating system for the server installation.
+
+Backend Features
+================
+* make the new full tree handling the default
+* Incremental storage of sources from upstream SCM systems
+  * cpio archives are stored incremental and offered the to the build job
+    as directory.
+  * The default git handling has been changed to use obs_scm service which
+    produces the right format.
```
Comment (adrianschroeter) [inline ReleaseNotes-2.7]: that is a webui/osc feature

File: ReleaseNotes-2.7 [hunk 3]
```diff
@@ -3,63 +3,126 @@
...
+* webui has support to add remote repositories
+* Instantiate package api call for projects building service packs
+* Support tracking of github issues
+* New option to avoid usage of defined repositories when branching a package
+  via the OBS:BranchSkipRepositories attribute
+* Groups can have an email adress for notifications and contact
+* New waitservice and mergeservice commands to handle _service files
+
+* make request numbering reliable when using a MySQL cluster
+* Lots of data input validations
```
Comment (bgeuken) [inline ReleaseNotes-2.7]: Are those 2 relevant here?

File: ReleaseNotes-2.7 [hunk 4]
```diff
@@ -3,63 +3,126 @@
...
+* use new obs_scm service for git source URLs
+* Make it possible to force Package.destroy
```
Comment (bgeuken) [inline ReleaseNotes-2.7]: That's the model method or something a user consciously can do?

File: ReleaseNotes-2.7 [hunk 5]
```diff
@@ -3,63 +3,126 @@
...
+  build type (request for build update to support it sent separately)
+* support buildtime source services
+  * skip them on service server
+  * add dependencies to build env
+
+
+Frontend Features
+=================
+* Support embargo of projects (#316776)
+* webui has support to add remote repositories
```
Comment (bgeuken) [inline ReleaseNotes-2.7]: Is that the DoD repository ui? If not I think that one is missing

**Comment by bgeuken** [comment]
Nothing I have a strong opinion about, but it looks like there are now a couple of changes mentioned that are probably just interesting for developers. Not sure if those are in the target group of a changelog

File: ReleaseNotes-2.7 (hunk 3 above)
Comment (adrianschroeter) [inline ReleaseNotes-2.7]: no, can be removed IMHO

File: ReleaseNotes-2.7 (hunk 4 above)
Comment (adrianschroeter) [inline ReleaseNotes-2.7]: It is a webui feature. We should seperate webui and api features here again so one gets the context.

**Comment by adrianschroeter** [comment]
I am about to "re-factor" this a bit, cherry-picked and reworking, another pull request will come.

**Comment by adrianschroeter** [comment]
It is here: https://github.com/openSUSE/open-build-service/pull/1755

**Comment by hennevogel** [comment]
Then lets close this :)

**Pull request closed without merging**